/**
 * @file publisher.hpp
 * @author Pooja Kabra
 * @brief This file publishes the data in string format
 * @version 0.1
 * @date 2021-11-08
 *
 * @copyright Copyright (c) 2021
 *
 */

#pragma once

#include <string>

/**
 * Structure to store the default display message 
 * Type of the message is string
 */
struct DefaultString {
  std::string default_msg = " Crazy cat lady ";
};
